xml/ay_networking.xml

```diff
@@ -19,51 +19,68 @@
  </info>
 
    <para>
-    Network configuration is used to connect a single workstation to an
-    Ethernet-based LAN or to configure a dial-up connection. More complex
-    configurations (multiple network cards, routing, etc.) are also
-    provided.
+    Network configuration is mainly used to connect a single workstation to an
+    Ethernet-based LAN and it is commonly configured even before starting &ay;
+    in order to fetch the profile from a network location. This network
+    configuration is usually done through <command>linuxrc</command>
    </para>
 
-   <para>
-    If the following setting is set to <literal>true</literal>, &yast;
-    will keep network settings created during the installation (via
-    <command>linuxrc</command>) and/or merge it with network settings from the
-    &ay; control file (if defined).
-   </para>
    <note>
     <title>The <command>linuxrc</command> program</title>
     <para>
      For a detailed description of how <command>linuxrc</command> works and its
      keywords, see <xref linkend="appendix-linuxrc"/>.
     </para>
    </note>
+
    <para>
-    &ay; settings have higher priority than any configuration files already
-    present. &yast; will write ifcfg-* files based on the entries in the control
-    file without removing old ones. If there is an empty or absent DNS and
-    routing section, &yast; will keep any pre-existing values. Otherwise,
-    settings from the control file will be applied.
+    By default &yast; copies the network settings that were used during the
+    installation to the finaly system. This configuration is merged with the
+    one defined in the &ay; profile. The copy of the configuration can be
+    ommited setting the <literal>keep_install_network</literal> option to
+    <literal>false</literal>.
    </para>
 
 <screen>&lt;keep_install_network
-config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
+config:type="boolean"&gt;false&lt;/keep_install_network&gt;</screen>
 
    <para>
-    During the second stage, installation of additional packages will take
-    place before the network, as described in the profile, is configured.
-    <literal>keep_install_network</literal> is set by default to
-    <literal>true</literal> to ensure that a network is available
-    in case it is needed to install those packages. If all packages
-    are installed during the first stage and the network is not needed early
-    during the second one, setting <literal>keep_install_network</literal>
-    to <literal>false</literal> will avoid copying the configuration.
+    &ay; settings have higher priority than any configuration files already
+    present. &yast; will write ifcfg-* files based on the entries in the profile
+    without removing old ones. If there is an empty or absent DNS and routing
+    section, &yast; will keep any pre-existing values. Otherwise, settings from
+    the profile file will be applied.
```

Comment:
```suggestion
    &ay; settings have higher priority than any existing configuration files.
    &yast; will write <filename>ifcfg-*</filename> files based on the entries in the profile
    without removing old ones. If the DNS and routing section is empty or missing,
    &yast; will keep any pre-existing values. Otherwise, it applies the settings from
    the profile file.
```

Reply:
In general, try to avoid using the passive voice (X will be...) unless necessary.